guides/common/modules/proc_filter-errata-by-type-severity.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Filter_errata_by_type_or_severity_{context}"]
+= Filter Errata by Type or Severity
+
+Use the following procedure to filter errata by type or severity.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts > All Hosts* and click the name of the host.
+. In the new hosts page click the *Contents* tab.
+. The *Errata* tab of the *Contents* page lists the errata associated with that host.
+. At the top of the page filter by Type by clicking the *Type* button.
```

Comment:
Please rephrase to something like "You can filter ..." or "Click the X button to filter...".